### DISCUSSION TITLE
Improving Movie Collections

### DISCUSSION THREAD

**Discussion by 4e576rt8uh9ij9okp** [jellyfin/jellyfin#10221] [Q&A]
Hello there, so I'm very unhappy how "good" collection handling is.
For example there is a folder with all the seasons inside but it doesn't create a collection for it.

So what I need, is to know the function that is responsible for handling collections.

**Comment by TheMelmacian** [comment]
Collections are only created automatically if you activate the "TheMovieDB Boxsets Plugin": https://github.com/jellyfin/jellyfin-plugin-tmdbboxsets. And it only creates collections for movies not TV-shows/serials.
But it's always possible to manually create a collection and to add any kind of item (movie, show, music album, book, other collection etc.) to a collection.

Since seasons of shows/serials are already grouped inside a show folder, there is no need to automatically create a collection for them.
If your seasons are not in the same parent folder, there is an option in the settings to group them inside a single show folder.

If you want additional automation for the creation of collections you need to implement a new plugin with the desired logic.
Or you can write a script (that for example uses the REST API) that creates the collections for you.

**Reply by GlassedSilver** [reply]
> Since seasons of shows/serials are already grouped inside a show folder, there is no need to automatically create a collection for them.

Ignoring anime which typically release successive seasons as separate shows I want to point out that spin-offs for Western media as well are typically separate shows rather than specials and therefore not connected to the main show in any meaningful way in your library.

Beyond that, it'd be nice to have collections and box sets as different structures altogether, because it'd be nice to group and visually connect in general prequels, sequels, spin-offs no matter the library type whilst having collections be more of a general grouping feature, maybe even with smart-filters so you could have Kometa (formerly known as Plex Meta Manager) kinds of collections without needing to run that additionally.

Such collections could then be used to liven up the home screen and recommendations with additional context like "Other Korean Movies" etc...

But even if that won't be integrated logic, no matter the source of creation of such collections based on criteria other than what belongs to the same franchise can truly elevate the library discoverability.

And again, movies ought to be groupable independent of the catch-all feature called Collections, that would also allow Movie 3 to appear as "sequel"-labeled recommendation on the Movie 2 page and Movie 1 appearing as "prequel". If you see that it's missing the watched mark you know what to watch next and probably didn't watch it yet because you forget the sequel and prequel have wildly different names.
e.g.: Peninsula and Train to Busan - QUICK! Without looking up, which one is the sequel and which one is the prequel?
Sure you can look at release years in Jellyfin's UI, but you know... there ought to be a more elegant way if Jellyfin still has that general goal of feeling like the proprietary privacy-invasive alternatives and services _(if only I could remember where I read that part, because it really resonated with me)_
